app/v1/models/request/createPeriodic/PeriodAllowableExpenses.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.request.createPeriodic
+
+import play.api.libs.functional.syntax.unlift
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+
+case class PeriodAllowableExpenses(consolidatedExpenses: Option[BigDecimal],
+                                   costOfGoodsAllowable: Option[BigDecimal],
+                                   paymentsToSubcontractorsAllowable: Option[BigDecimal],
+                                   wagesAndStaffCostsAllowable: Option[BigDecimal],
+                                   carVanTravelExpensesAllowable: Option[BigDecimal],
+                                   premisesRunningCostsAllowable: Option[BigDecimal],
+                                   maintenanceCostsAllowable: Option[BigDecimal],
+                                   adminCostsAllowable: Option[BigDecimal],
+                                   businessEntertainmentCostsAllowable: Option[BigDecimal],
+                                   advertisingCostsAllowable: Option[BigDecimal],
+                                   interestOnBankOtherLoansAllowable: Option[BigDecimal],
+                                   financeChargesAllowable: Option[BigDecimal],
+                                   irrecoverableDebtsAllowable: Option[BigDecimal],
+                                   professionalFeesAllowable: Option[BigDecimal],
+                                   depreciationAllowable: Option[BigDecimal],
+                                   otherExpensesAllowable: Option[BigDecimal])
+
+object PeriodAllowableExpenses {
+  implicit val reads: Reads[PeriodAllowableExpenses] = Json.reads[PeriodAllowableExpenses]
+
+  implicit val writes: OWrites[PeriodAllowableExpenses] = (
```

Comment:
would it be possible to call this something like `downstreamWrites` to clarify its use?
(same with the others)